doc/libfuzzer.md

```diff
@@ -0,0 +1,57 @@
+# libfuzzer
+
+## setup 
+
+### Requirements
+libFuzzer needs LLVM sanitizer support, 
+so this only works on x86-64 Linux, 
+x86-64 macOS and Apple-Silicon (aarch64) macOS for now. 
+This also needs a nightly compiler since 
+it uses some unstable command-line flags. 
+You'll also need a C++ compiler with C++11 support.
+
+
+
+### Installing
+
+rust-version|libfuzzer|td-shim|result
+:-:|:-:|:-:|:-:
+nightly-2020-02-08|0.11.0|staging|no (td-shim fails to compile in this version)
+nightly-2021-08-20|0.10.2|staging|yes
+nightly-2020-02-08|0.11.0|staging|no (libfuzzer failed to start)
+
+```
+cargo install cargo-fuzz --version 0.10.2
+# coverage
+rustup component add llvm-tools-preview rust-src
+cargo install grcov
+```
+### Provide starting inputs
+
+Use RAMdisks for input since, we don't want to destroy harddrives
+
+```
+$sudo mount -t tmpfs -o size=1024M tmpfs fuzz/corpus
```

Comment:
Consistency: some commands have a `$ ` shell prompt prefix and some don't. Kata strongly encourages a shell prompt for all commands so you may want to adopt that approach for clarity. See below for details:

- https://github.com/kata-containers/kata-containers/blob/main/docs/Documentation-Requirements.md#code-blocks

Reply:
done